i18n/fr.json

```diff
@@ -1306,8 +1306,8 @@
   "admin.logs.prev": "Précédent",
   "admin.logs.reload": "Recharger",
   "admin.logs.title": "Journaux du serveur",
-  "admin.manage_roles.additionalRoles": "Sélectionnez les autorisations additionnelles pour le compte. Pour en savoir plus sur les rôles et les autorisations, [consultez notre documentation](!https://about.mattermost.com/default-permissions).",
-  "admin.manage_roles.allowUserAccessTokens": "Autoriser ce compte à générer des [jetons d'accès personnels](!https://about.mattermost.com/default-user-access-tokens).",
+  "admin.manage_roles.additionalRoles": "Sélectionnez les autorisations additionnelles pour le compte. Pour en savoir plus sur les rôles et les autorisations, [consultez notre documentation](!https://developers.mattermost.com/integrate/admin-guide/admin-personal-access-token/).",
```

Comment:
@cwarnermm This one also seems like it might have been incorrect since it was changed from a page on permissions to one on personal access tokens